app/views/spaces/_header.html.erb

```diff
@@ -0,0 +1,7 @@
+<div style='background-color: <%= current_space.brand_color %> '>
+  <h1><%= current_space.name %></h1>
+  <% policy_scope(current_space.rooms).each do |room| %>
+    <%= link_to room.name, polymorphic_path(room.location), id: dom_id(room, :link_to), class: "group no-underline" %>
+  <% end %>
+</div>
+</div>
```

Comment:
This would be marked up and styled like the Moment Park header as a sensible default.